### PR TITLE
Python code formatting on the codebase implemented except "line too long error"

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
### Description
1.import * has been removed.
2.whitespacing errors resolved.
3.Unused modules are removed from import except for the tests.
4.Proper indentation done.
All the errors have been fixed except for line too long

Fixes # https://github.com/systers/vms/issues/537

### Type of Change:
- Code

### How Has This Been Tested?
screenshot from 2018-01-18 12-38-10

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: vms/volunteer/views.py [hunk 1]
```diff
@@ -41,7 +41,9 @@ def download_resume(request, volunteer_id):
                 filename = settings.MEDIA_ROOT + basename
                 wrapper = FileWrapper(file(filename))
                 response = HttpResponse(wrapper)
-                response['Content-Disposition'] = 'attachment; filename=%s' % os.path.basename(filename)
+                response[
+                    'Content-Disposition'] = 'attachment; filename=%s' % os.path.basename(
```
Comment (tapasweni-pathak) [inline vms/volunteer/views.py]: this doesn't look good, what was the length w/ 45 being w/ 44? we can think of increasing the length to 85.

File: requirements.txt [hunk 2]
```diff
@@ -11,3 +11,4 @@ phonenumbers==7.2.6
 django-cities-light==3.2.0
 Unidecode==0.4.19
 django-braces==1.9.0
+pycodestyle==2.3.1
```
Comment (tapasweni-pathak) [inline requirements.txt]: wouldn't `flake8` would do, do we need `pycodestyle` if we have `flake8`? 🤔

Comment (anjali-dhanuka) [reply requirements.txt]: Yes flake8 will do. I have removed pycodestyle.

File: vms/volunteer/views.py (hunk 1 above)
Comment (anjali-dhanuka) [reply vms/volunteer/views.py]: All your suggested changes have been implemented. Please have look

Comment (anjali-dhanuka) [reply vms/volunteer/views.py]: All your suggested changes have been implemented now. Most of the Flake8 errors have also been solved. It now also fixes the issue https://github.com/systers/vms/issues/379. Please have a look.

Comment (tapasweni-pathak) [reply vms/volunteer/views.py]: Can you please resolve the conflicts too, you would see that `flake8 .` in `.travis.yml` is you need to add as it was recently removed.

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak I have resolved the conflicts. Please review.

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak  Done with resolving conflicts. Please review

**Comment by anjali-dhanuka** [comment]
@tapasweni-pathak done with rebasing. And for testing purposes I ran flake8 and noticed that the warnings were greatly reduced.

**Pull request merged**